website/docs/cloud-docs/registry/publish-providers.mdx

```diff
@@ -137,7 +137,7 @@ $ curl \
   --header "Content-Type: application/vnd.api+json" \
   --request POST \
   --data @payload.json \
-  https://app.terraform.io/api/v2/organizations/ORG_NAME/registry-providers/private/ORG_NAME/aws/versions
+  https://app.terraform.io/api/v2/organizations/ORG_NAME/registry-providers/private/ORG_NAME/PROVIDER_NAME/versions
```

Comment:
So the below the response also uses AWS (`-T terraform-provider-aws_5.14.0_SHA256SUMS \`)— maybe instead of making this generic, we could callout that this is an example using the AWS provider above? 

Something like, "Replace `TOKEN` in the `Authorization` header with your HCP Terraform API token, and replace both instances of `ORG_NAME` with your organization's name. **If are not using the `aws` provider, then replace `aws` with your provider's name.**"

Reply:
Hey Rose, the first one was typo, sorry. Additionally, regarding the second suggestion, I agree with you it sounds much better, should I edit it and commit or you will proceed with doing so?

Reply:
Can you edit it and commit please?